### PR TITLE
update blog menu and files

### PR DESCRIPTION
**Pull request by csantanapr**
Update blog files

- sync images
- update readme
- update nav to match main site

### CODE REVIEW

File: blog/README.md [hunk 1]
```diff
@@ -34,15 +33,16 @@ The main one, for the website as a whole, is located at [`docs/config/nav.yml`](
 
 Note here that each link assumes that the present working directory is `docs/docs/`, so for example the "Getting Started" README which is located in `docs/docs/getting-started/README.md` is listed as `getting-started/README.md`
```
Comment (snneji) [inline blog/README.md:34]: Do we also want to update "Getting Started" to "Tutotial" here? Also, maybe update the snippet in lines 25 - 32 to match the current `nav.yml`.

File: blog/README.md [hunk 2]
```diff
@@ -5,15 +5,14 @@ The Knative website has a basic, top-level navigation that looks like this:
...
     - Installing:
     - Serving:
     - Eventing:
     - Code samples:
     - Reference:
-    - About:
     - Blog:
-    - Case studies:
+    - About:
     - Community:
```
Comment (snneji) [inline blog/README.md:16]: Currently the order in this snippet matches the order we can see on the site (Blog, About, Community), but should this match the order in the `nav.yml` (Community, About, Blog)?

**Review by snneji — COMMENTED** [review]
Thanks for syncing these. I forgot to do it when I changed the getting started section!

I have just a few questions.

File: blog/README.md (hunk 2 above)
Comment (csantanapr) [reply blog/README.md:16]: For some reason the last items are reversed on the site from what is in nav.yaml

I will update to match nav.yaml

**Comment by csantanapr** [comment]
@snneji PITAL again when you have a chance

**Pull request merged**